storage/storage_setup.py

```diff
@@ -19,57 +19,37 @@ def schema_setup(ip_addresses, username, password, keyspace):
 
     #Create the Knowledge Base tables and related Materialized Views
     session.execute(
-        """CREATE TABLE IF NOT EXISTS object_knowledge_base_table(
+        """CREATE TABLE IF NOT EXISTS knowledge_base_table(
```

Comment:
when this is merged into the bigger project, I would recommend making this something that clearly defines what the table is related to. For example analytics_relationship_features_base_table